### ISSUE TITLE
Timestamps should probably be millisecond resolution by default

### ISSUE THREAD

**Issue by bitemyapp** [Soostone/katip#92]
Here's some code a user provided of their variation:

```haskell
multiscribeEnvironment backends nameSpace env = do
  logEnv <- initLogEnv nameSpace env

  let timedLogEnv = logEnv { _logEnvTimer = getCurrentTime }
      registerScribe' (LogBackend name logLevel st hdl) = registerScribe
        name
        (scribeConstructor st (fromLogLevel logLevel) defaultVerbosity hdl)
        defaultScribeSettings
  -- by default Katip uses an AutoUpdate function with a 1 second interval update, which
  -- means logs's precision is 1 second max
  logEnv' <- foldM (flip registerScribe') timedLogEnv backends
  pure $ LoggerEnv Map.empty logEnv' mempty
```

**Comment by MichaelXavier** [comment]
Released with `katip-0.5.5.1`

**Comment by bitemyapp** [comment]
@MichaelXavier thank you!
